requirements.txt

```diff
@@ -1,4 +1,5 @@
 lxml
 scipion-pyworkflow
 scipion-em
-scipion-chem
\ No newline at end of file
+scipion-chem
+scikit-learn
```

Comment:
how are we finally going to treat these requirements that are not scipion?

Reply:
They should be deleted from this file, and, wherever they are being imported and used, move such code to a separate script and call that script with
```Python
protocol.runScript("myscript.py", env="myEnvContainingTheLibrary")
```
For that, the steps would be:

- Create a new enviroment in the installation or reuse an existing one to include such library.
- Move the code to the script and call that script from the protocol or viewer using that env.
- Remove the library dependency from this file.